### PR TITLE
Allow to disable auto open new tab when starting dev server

### LINKED ISSUE #134: Disable auto open new tab when starting dev server
It looks like there is no way to overwrite this 

https://github.com/MoOx/statinamic/blob/master/src%2Fbuilder%2Fserver.js#L21

Why? It just too annoying for me

### PR DESCRIPTION
**Pull request by thangngoc89**
This will close #134 
Could you cut a release after merging this ? You know. It just too annoying.

### CODE REVIEW

File: src/configurator/index.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const defaultOptionsCLIonly = {
   production: false,
   static: false,
   server: false,
+  open: false,
```
Comment (MoOx) [inline src/configurator/index.js]: should be "true" by default. Otherwise it's a breaking change, not a fix :)

Comment (MoOx) [inline src/configurator/index.js]: Also maybe this option should not be "cli only" so we can adjust from package.json

Comment (thangngoc89) [inline src/configurator/index.js]: Alright, so this should be disabled via package.json. but can also be enabled via cli

**Comment by thangngoc89** [comment]
`open` option should be able to disabled via package.json. but can also be able to enabled via cli.

**Comment by MoOx** [comment]
Can you add it in the Configuration doc section? Ad use "Added:" not "Fixed:" in CHANGELOG.

**Comment by thangngoc89** [comment]
Should be good now

**Comment by MoOx** [comment]
Will merge when merging "next" branch (this PR scope is master).

**Comment by thangngoc89** [comment]
:+1:

**Comment by MoOx** [comment]
Conflict so I cherry picked/updated the commit directly in master.

**Pull request closed without merging**